### PR TITLE
Service Bus Track2 - Fixes and Samples for using AAD/application id

### PR DESCRIPTION
**Pull request by hemanttanwar**
This PR has following features.

1. Fixes issue with AAD when using management channel
2. Sample showing how AAD can be used for send and receive

fixex #8711

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessage.java [hunk 1]
```diff
@@ -159,15 +159,6 @@ public ServiceBusMessage(String messageId, byte[] body, String contentType) {
...
     }
 
-    /**
-     * Returns message body as UTF-8 decoded string.
-     *
-     * @return UTF-8 decoded string representation of the message.
-     */
-    public String getBodyAsString() {
-        return new String(body, UTF_8);
-    }
```
Comment (hemanttanwar) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessage.java]: This should be removed form API

File: sdk/servicebus/azure-messaging-servicebus/README.md [hunk 2]
```diff
@@ -76,22 +83,80 @@ platform. First, add the package:
...
 #### Authorizing with AAD application client secret
 
 Follow the instructions in [Creating a service principal using Azure Portal][application_client_secret] to create a
 service principal and a client secret. The corresponding `clientId` and `tenantId` for the service principal can be
 obtained from the [App registration page][app_registration_page].
 
-When using Azure Active Directory, your principal must be assigned a role which allows access to Event Hubs, such
-as the `Azure Event Hubs Data Owner` role. For more information about using Azure Active Directory authorization
-with Event Hubs, please refer to [the associated documentation][aad_authorization].
+When using Azure Active Directory, your principal must be assigned a role which allows access toService Bus, such
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/README.md]: -> `to Service Bus`

File: sdk/servicebus/azure-messaging-servicebus/README.md [hunk 3]
```diff
@@ -25,6 +25,13 @@ have to be online at the same time.
   - [Authenticate the client](#authenticate-the-client)
 - [Key concepts](#key-concepts)
 - [Examples](#examples)
+  - [Create a sender or receiver using connection string](#create-a-sender-or-receiver-using-connection-string)
+  - [Send Message to Queue or Topic](#send-message-to-queue-or-topic)
+  - [Receive message from Queue or Subscription](#receive-message-from-queue-or-subscription)
+  - [Peek message from Queue or Subscription](#peek-message-from-queue-or-subscription)
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/README.md]: Send and receive using AAD and connection string are good scenarios. We don't need to duplicate it with peek (this is also a diagnostic scenario we don't want to put to the forefront) and other small operations. 

It makes the table of contents long. (Feedback we received in EH)

File: sdk/servicebus/azure-messaging-servicebus/README.md [hunk 4]
```diff
@@ -76,22 +83,80 @@ platform. First, add the package:
...
 
-When using Azure Active Directory, your principal must be assigned a role which allows access to Event Hubs, such
-as the `Azure Event Hubs Data Owner` role. For more information about using Azure Active Directory authorization
-with Event Hubs, please refer to [the associated documentation][aad_authorization].
+When using Azure Active Directory, your principal must be assigned a role which allows access toService Bus, such
+as the `Azure Service Bus Data Owner` role. For more information about using Azure Active Directory authorization
+with Service Bus, please refer to [the associated documentation][aad_authorization].
 
 ## Key concepts
+#### Queues
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/README.md]: Should be 3rd level headers, not fourth.

**Pull request merged**